src/develop/blend_gui.c

```diff
@@ -1961,7 +1961,7 @@ static GtkWidget *_combobox_new_from_list(dt_iop_module_t *module, const gchar *
   dt_bauhaus_widget_set_label(combo, _("blend"), label);
   gtk_widget_set_tooltip_text(combo, tooltip);
   for(; *list->name; list++)
-    dt_bauhaus_combobox_add_full(combo, _(list->name), DT_BAUHAUS_COMBOBOX_ALIGN_RIGHT,
```

Comment:
This one was correct and necessary I'm afraid. A few drop downs in blending are now not being translated.

Reply:
I don't understand how is this supposed to work, maybe I'm wrong ! If you revert just this the text is translated again?

Reply:
Yes.

_() is an alias for gettext. It serves two purposes; the scanner looks for a quoted text between the brackets and if it finds it, will add it to the translation list. At runtime, this will simply translate whatever parameter it is fed; that doesn't have to be a literal.

Reply:
Stupid me, got confused then :( Sorry let me know if you want me to revert this part or if you already have done that.

Reply:
if you wouldn't mind reverting? it leaves a cleaner  record and I'm in the middle of a large rebase (that's how I came across it) that I don't want to mess up (too  much). Thank you!

Reply:
Done, thanks for catching!